meta-iot2000-example/recipes-core/images/iot2000-example-image.inc

```diff
@@ -48,3 +48,5 @@ DEPENDS += "gptfdisk-native efibootguard-native"
 
 # The following is needed for correct EFI deployment
 do_image_wic[depends] += "${PN}:do_bootimg"
+
+IMAGE_INSTALL_append = " persiststore"
```

Comment:
Well, while nice, what's the reason to include this unconditionally?

Reply:
Because this leads to a minimal diff between no-swupdate / swupdate version. These are example images as the name says. Nobody is forced to use it. Tell me a reason to include it conditionally.

Reply:
That's a reason, fine with me :)